### ISSUE TITLE
Only one custom inline style is persisted at a time when text is not selected

### ISSUE THREAD

**Issue by soniapatel** [facebook/draft-js#1690]
**Do you want to request a *feature* or report a *bug*?**
bug

**What is the current behavior?**
Only one custom inline style is persisted at a time when text is not selected.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem. You can use this jsfiddle to get started: 
After selecting text, any custom inline style can be applied from font style, font size, text color or background color.
With selecting text if more than one custom inline style are selected, it resets the previously selected style to the default value.
We are using toggleCustomInlineStyle from the draft-utils.
![200w_d](https://user-images.githubusercontent.com/1992927/37427728-7a4123e2-2787-11e8-978f-3df2ea79be7e.gif)

**What is the expected behavior?**
All the custom inline styles should be persisted and applied to the text that the user types in the editor.

**Which versions of Draft.js, and which browser / OS are affected by this issue? Did this work in previous versions of Draft.js?**
"draft-js": "^0.10.4"
Mac: Chrome, Firefox.
It did not work in any version of draft js.

**Comment by thibaudcolas** [comment]
Hey @soniapatel, it's very hard to tell from the super small GIF but it looks like you're using https://github.com/jpuri/react-draft-wysiwyg. 

I think this is most likely an issue with that particular project, not Draft.js. On the Draft.js-only https://draftjs.org/ editor, you can correctly select text and then apply multiple inline styles.

**Comment by kamenminkovcom** [comment]
@thibaudcolas check [this one](https://codepen.io/kamenminkovcom/pen/WBdGmV) The behavior is the same as @soniapatel describes. I have also created an [issue](https://github.com/facebook/draft-js/issues/2079) and think that the issues are pretty much the same so feel free to close it.

**Comment by thibaudcolas** [comment]
Yes, this does sound like the same issue. This is working as I would expect in my own editor and on https://draftjs.org/, so yep I wouldn't be surprised if this was just a matter of using `onMouseDown` instead of `onClick` as you describe so the selection isn't reset in-between clicks.
